assets/stylesheets/sass/main/main.scss

```diff
@@ -0,0 +1,164 @@
+@import "bourbon/bourbon";
+@import "components/colors";
+@import "components/typography";
+
+body {
+  font-size: 100%;
+  line-height: 1;
+  margin: 0;
+  padding: 0;
+}
+
+aside,
+div,
+header {
+  box-sizing: border-box;
+}
+
+.button {
+  background-color: $brand-blue;
+  border-radius: 0.25em;
+  box-shadow: none;
+  color: $white;
+  display: inline-block;
+  font-weight: 600;
+  padding: 0.8em 1.3em;
+  text-decoration: none;
+
+  &:active {
+    background-color: $brand-blue-darkest;
+  }
+
+  &:hover {
+    background-color: $brand-blue-darker;
+  }
+}
+
+.close {
+  color: $light-gray;
+  cursor: pointer;
+  font-family: "Helvetica", sans-serif;
+  font-size: 1.65em;
+  font-weight: 400;
+  line-height: 1em;
+  position: absolute;
+  right: 0.5em;
+  top: 0.5em;
+}
+
+.hero {
+  background-color: $light-gray;
+  color: $white;
+  height: 600px;
+  padding-top: 6em;
+
+  h1 {
+    font-size: 4em;
+    margin-bottom: 0.25em;
+  }
+
+  p {
+    line-height: 1.5em;
+    margin-top: 0;
+  }
+
+  .content {
+    margin: 0 auto;
+    max-width: 30em;
+    text-align: center;
+  }
+}
+
+.js {
+  .no-js-hide {
+    display: none;
+  }
+}
+
+.modal {
+  @include transform(translateY(10%));
+  @include transition(all 0.4s ease);
+  background: $white;
+  border: 2px solid $light-gray;
+  box-shadow: 0 3px 3px rgba($dark-gray, 0.1);
+  box-sizing: border-box;
+  height: 350px;
+  left: 50%;
+  margin: -175px 0 0 -250px;
+  opacity: 0;
+  padding: 2.25em 1.5em 1.5em;
+  position: absolute;
+  top: 50%;
+  visibility: hidden;
+  width: 500px;
+  z-index: 1000;
+
+  &.active {
+    @include transform(translateY(0));
+    opacity: 1;
+    visibility: visible;
+  }
+
+  h2 {
+    color: $brand-blue-darker;
+    text-align: center;
+  }
+
+  p {
+    line-height: 1.3em;
+    text-align: justify;
+  }
+
+  .action {
+    text-align: center;
+  }
+
+  .button {
+    font-size: 1.1em;
+    margin-top: 1.25em;
+  }
+}
+
+.state-select {
+  margin-top: 2em;
+
+  select {
+    @include appearance(none);
+    background: $white url("../../../images/arrow-down.svg") no-repeat;
+    background-position: right 1.3em center;
+    background-size: 1em 1em;
+    border: 1px solid #5b616b;
+    border-radius: 0;
+    box-sizing: border-box;
+    color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.